app/models/claim/advocate_claim.rb

```diff
@@ -179,8 +179,9 @@ def eligible_basic_fee_types
       # TODO: this should return a list based on the current given fee scheme
       # rather than conditionally return scheme 10 specifically
       # TBD once all the fee scheme work is integrated
-      return Fee::BasicFeeType.agfs_scheme_9s unless fee_scheme == 'fee_reform'
-      Fee::BasicFeeType.unscoped.agfs_scheme_10s.order(:position)
+      return Fee::BasicFeeType.all if from_api? || from_json_import?
```

Comment:
I now official hate anything where I see `from_api?` in it 😅 😭

Reply:
know what you mean.